crates/uv/src/commands/python/install.rs

```diff
@@ -41,10 +44,15 @@ pub(crate) async fn install(
 
     let targets = targets.into_iter().collect::<BTreeSet<_>>();
     let requests: Vec<_> = if targets.is_empty() {
-        PythonVersionFile::discover(project_dir, no_config, true)
-            .await?
-            .map(PythonVersionFile::into_versions)
-            .unwrap_or_else(|| vec![PythonRequest::Default])
+        PythonVersionFile::discover(
+            project_dir,
+            &VersionFileDiscoveryOptions::default()
+                .with_no_config(no_config)
+                .with_preference(VersionFilePreference::Versions),
```

Comment:
Why do we prefer `.python-versions` here but nowhere else?

Reply:
Installation can operate on multiple versions, while the other operations require a single version. This is the existing behavior, just clearly written here.